ansible/roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -116,6 +116,13 @@ data "openstack_networking_network_v2" "cluster_external_network" {
   name = "{{ cluster_external_network }}"
 }
 
+# SRIOV storage network 
+{% if cluster_storage_network is defined %}
+data "openstack_networking_network_v2" "cluster_storage_direct" {
```

Comment:
lets just call this cluster_storage for now, so we match the cluster_storage_network parameter please?